instances/widgets.treasury-factory.near/widget/pages/asset-exchange/OneClickExchangeFormIframe.jsx

```diff
@@ -703,8 +703,14 @@ const code = `
                             <div class="collapse-item">
                                 <div class="detail-label">
                                     <span>Estimated time</span>
+                                    <i class="bi bi-info-circle text-secondary"
+                                       data-bs-toggle="tooltip"
+                                       data-bs-custom-class="custom-tooltip"
+                                       data-bs-placement="top"
+                                       title="Estimated time for the swap to be executed after the deposit transaction is confirmed">
+                                    </i>
```

Comment:
so technically the deposit happens when the approval is reached so may be we should do something like 
```suggestion
                                       title="Estimated time for the swap to be executed after the proposal is approved and the amount is deposited">
                                    </i>
```